server/views/resources/how-nonprofit-projects-work.jade

```diff
@@ -0,0 +1,47 @@
+extends ../layout
+block content
+    .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+        html.
+            <h2 class='text-center'>How our Nonprofit Projects Work</h2>
+            <hr>
```

Comment:
Because we're getting ready to move to React and there's no point in using Jade if we'll just have to move it back to HTML. I've been using HTML in a lot of the text heavy static views for this reason.